Roadmap.md

```diff
@@ -0,0 +1,135 @@
+# Roadmap
+
+This document outlines the roadmap of the Mixed Reality Toolkit.
+
+# Upcoming Releases
+
+The following table lists the releases planned through the end of 2018 for the Mixed Reality Toolkit. Release details, including backlog items, can be found on the [GitHub project pages](https://github.com/Microsoft/MixedRealityToolkit-Unity/projects).
+
+| Product | Version | Timeline | Project board |
+| --- | --- | --- | --- |
+| [Mixed Reality Toolkit](#mixed-reality-toolkit-mrtk-roadmap) | 2018.7.0.0 (Alpha) | July 2018 | [Version Next Alpha](https://github.com/Microsoft/MixedRealityToolkit-Unity/projects/11) |
+| [Mixed Reality Toolkit](#mixed-toolkit-toolkit-roadmap) | 2018.8.0.0 (Beta) | August 2018 | [Version Next Beta](https://github.com/Microsoft/MixedRealityToolkit-Unity/projects/12) |
+| [Mixed Reality Toolkit](#mixed-reality-toolkit-roadmap) | 2018.9.0.0 | September 2018 | [2018.9.0.0 (Version Next initial release)](https://github.com/Microsoft/MixedRealityToolkit-Unity/projects/14) |
+| [Mixed Reality Toolkit](#mixed-toolkit-toolkit-roadmap) | 2018.10.0.0 | October 2018 | |
+| [Mixed Reality Toolkit](#mixed-toolkit-toolkit-roadmap) | 2018.11.0.0 | November 2018 | |
+
+Release details, including backlog items, can be found on the [GitHub project pages](https://github.com/Microsoft/MixedRealityToolkit-Unity/projects).
+
+# Mixed Reality Toolkit (MRTK) Roadmap
+
+The Mixed Reality Toolkit (also known as "vNext") is an all-new product, built to be cross MR/AR/VR/XR platform by design. There are two planned pre-releases after which the Mixed Reality Toolkit will become the primary product.
+
+The Mixed Reality Toolkit will require Unity 2018 or newer.
+
+| When Unity releases an LTS (Long Term Support) product, the Mixed Reality Toolkit will branch (ex: mrtk_2018_LTS) and require an LTS release. This branch will be targeted at developers who wish to have the most stable MRTK on which to build. It is expected that the branched Mixed Reality Toolkit will receive bug fixes and no new features. The mrtk_development branch will update to the next Unity Tech release (ex: 2019.1) and **may** drop backward compatibility.
```

Comment:
Is the "|" meant to be a ">" character to denote a blockquote?
Change or remove.

Reply:
It wasn't intended to be a blockquote, but I agree that is a better format for this.